docs/detections/api/lists/api-import-list-items.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[lists-api-import-list-items]]
 === Import list items
 
-Imports a list of items from a `.txt` or `.csv` file.
+Imports a list of items from a `.txt` or `.csv` file. The maximum file size is 9K bytes. There is no minimum file size limit.
```

Comment:
```suggestion
Imports a list of items from a `.txt` or `.csv` file. The maximum file size is 9 KB. There is no minimum file size limit.
```

Reply:
Hey @nastasha-solomon @yctercero I did a mistake in terms of the file size. It is actually `9M bytes` instead of `9K bytes` https://github.com/elastic/kibana/blob/main/x-pack/plugins/lists/server/config.ts#L25. I'm not sure how we would like to present it better in docs, but saying that `9.000.000 bytes` is equal to `9MB` will be wrong in this case, since `9MB == 9.437.184 bytes`

Reply:
Based on [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/numbers) ("a round number of 1 million or more"):
```suggestion
Imports a list of items from a `.txt` or `.csv` file. The maximum file size is 9 million bytes. There is no minimum file size limit.
```